tests/test_add_parent_mass.py

```diff
@@ -33,6 +34,25 @@ def test_add_parent_mass_no_pepmass():
     assert numpy.abs(spectrum.get("parent_mass") - 445.0) < .01, "Expected parent mass of about 445.0."
 
 
+@pytest.mark.parametrize("adduct, expected", [("[M+2Na-H]+", 399.02884),
+                                              ("[M+H+NH4]2+", 212.47945),
+                                              ("[2M+FA-H]-", 843.001799)])
+def test_add_parent_mass_using_adduct(adduct, expected):
+    """Test if parent mass is correctly derived from adduct information."""
+    mz = numpy.array([], dtype='float')
+    intensities = numpy.array([], dtype='float')
+    metadata = {"precursor_mz": 444.0,
+                "adduct": adduct,
+                "charge": +1}
+    spectrum_in = Spectrum(mz=mz,
+                           intensities=intensities,
+                           metadata=metadata)
+
+    spectrum = add_parent_mass(spectrum_in)
+
+    assert numpy.allclose(spectrum.get("parent_mass"), expected, atol=1e-4), "Expected parent mass of about 339.0."
```

Comment:
Parent mass is different for each parameterize item.
```suggestion
    assert numpy.allclose(spectrum.get("parent_mass"), expected, atol=1e-4), f"Expected parent mass of about {expected}."
```